### DISCUSSION TITLE
Allow option to move using W, A, S, and D without needing to hold right click.

### DISCUSSION THREAD

**Discussion by G01dY05h1** [Mojang/minecraft-editor#77] [Feedback]
This would allow a user to move around a little easier and would stop a user from (for example) not moving if they push "W" before holding right click.

This would not apply if a user is typing in a text box or holding "ctrl" (in case a keybind is assigned to one of the 4 keys)

**Comment by masonfaulk2008** [comment]
It works the way it does for reasons, they implemented crosshair mode. It’s up to you to learn the ropes

**Comment by minecraftchloe** [comment]
There's a ton of complexity with the UI over the viewport and then allowing for the free-moving mouse to interact on top of it -- Crosshair mode is your best bet for this!
